tools/sdk-testgen/packages/autorest.testmodeler/README.md

```diff
@@ -67,20 +67,39 @@ The default value of this option is true.
 
 ### --testmodeler.use-example-model
 
-This options switch whether ExampleModel in generated in test model. While default to be true, it can be disabled like below:
+This option switch whether ExampleModel in generated in test model. While default to be true, it can be disabled like below:
 ```
 testmodeler:
     use-example-model: false
 ```
 
 ### --testmodeler.add-armtemplate-payload-string
 
-This options switch whether StepArmTemplateModel.armTemplatePayloadString is added into testmodeler. While default to be false, it can be enabled like below:
+This option switch whether StepArmTemplateModel.armTemplatePayloadString is added into testmodeler. While default to be false, it can be enabled like below:
 ```
 testmodeler:
     add-armtemplate-payload-string: true
 ```
 
+### --testmodeler.api-scenario-loader-option
+
+The api-scenarios are loaded from the autorest input-files by default. This option provide a gate to load api-scenario from other remote/branch/commit.
+This option are passed through directly to oav scenario loader, refer to https://github.com/Azure/oav/blob/develop/lib/apiScenario/apiScenarioLoader.ts#L60 for more detail of it. A sample for this option:
+```
+testmodeler:
+    api-scenario-loader-option:
+        fileRoot: https://github.com/Azure/azure-rest-api-specs/blob/eb829ed4739fccb03dd2327b7762392e74c80ae4/specification/appplatform/resource-manager
```

Comment:
I am not quite understand this. The name is related to api-scenario, but detailed configuration is about the swagger file paths.  Is this expected?